src/Dockerfile.subscription

```
@@ -0,0 +1,17 @@
+# FIX VERSION at release time:
+ARG BONITA_VERSION=8.0-SNAPSHOT
+
+FROM bonitasoft.jfrog.io/docker-snapshot-local/bonita-subscription:$BONITA_VERSION
```

To begin with:
```suggestion
FROM bonitasoft.jfrog.io/docker-releases/bonita-subscription:$BONITA_VERSION
```

And when we will open JFrog to all customers it will be `bonitasoft.jfrog.io/docker` repository.

---

Sure, but to test the current 8.0 (the one with the appropriate runtime), we need to point to the snapshot repository